packages/strapi-admin/admin/src/containers/Admin/index.js

```diff
@@ -144,6 +144,7 @@ export class Admin extends React.Component {
 
     return (
       <GlobalContextProvider
+        autoReload={this.props.global.autoReload}
```

Comment:
`this.props.global` is already destructured in the render method. You should destructure `this.props.global.autoReload`  too.

Reply:
You are right